### DISCUSSION TITLE
Profile Readme localization

### DISCUSSION THREAD

**Discussion by Marsgames** [community/community#59560] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hey,

I think profile readme pages are an incredible thing to highlight some information / project to anyone that will visit our page.
The only thing that bothers me is that we can only have one page showed.

I think for many users, mostly English speakers it's not a big issue, but as a French speaker, I would love to being able to have my profile page in both English and French. 

Most of the users that will find my profile are French recruiters or French companies. For this reason, my profile should be in French, I guess.
But I have some public projects localized and used by non-French speakers. So, I don't want to exclude them when they find my profile by having my page only in French. Even more if some of my projects relies on non-French users, and I need them to provide feedbacks, discussions, and more.

Wouldn't it be possible to have multiple readmes (like readme.md would be the default one, readme_fr.md would be the French one, readme_it.md Italien one, and so on...)
And with the browser locale, GitHub website would show the right or default readme, and user would still be able to switch to any available language.

I think that can be a great improvement.

Please let me know if it's planned or in the contrary it will never be implement for any reason.

Regards,
Marsgames.

**Comment by roma-marshall** [comment]
You can create several readme repositories and translate them into other languages. Keep the links to the other repositories in your main readme repository. This is the only way to localize at the moment. You can also create your own personal github pages and write your own localizations for any languages.
